app/portainer/services/api/userService.js

```diff
@@ -157,6 +136,40 @@ angular.module('portainer.app').factory('UserService', [
       return deferred.promise;
     };
 
+    service.createAccessToken = function (id, description) {
+      const deferred = $q.defer();
+      const payload = { description };
+      Users.createAccessToken({ id }, payload)
+        .$promise.then((data) => {
+          deferred.resolve(data);
+        })
+        .catch(function error(err) {
+          deferred.reject({ msg: 'Unable to create user', err: err });
+        });
+      return deferred.promise;
+    };
+
+    service.getAccessTokens = function (id) {
+      var deferred = $q.defer();
```

Comment:
same here, use async/await

Reply:
(same reason as above - retain code familiarity)

Reply:
well, code familiarity is subjective, I think most of the team won't understand this code. since QA is testing it anyway I think we should refactor what we can. I will approve the PR anyway